lib/base_service.js

```diff
@@ -31,13 +32,48 @@ class BaseService {
         this._logger = null;
         this._metrics = null;
         this._ratelimiter = null;
-        this.serviceReturns = null;
+        this._serviceReturns = null;
+        this._uncaughtExceptionHandlerAdded = false;
+        this._unhandledRejectionHandlerAdded = false;
```

Comment:
We don't really need these booleans, we can just initialize the handlers to underfined.

Reply:
Done